__tests__/app.test.js

```diff
@@ -40,13 +52,47 @@ describe('/api/topics', () => {
     })
 })
 
-describe('/api/not-a-route', () => {
-    test('GET:404 when endpoint does not exist', () => {
+describe('/api/articles/:article_id', () => {
+    test('GET:200 responds with an array of an article', () => {
         return request(app)
-        .get('/api/not-a-route')
-        .expect(404)
+        .get('/api/articles/1')
+        .expect(200)
         .then(({body}) => {
-            expect(body.msg).toBe('Route Not Found')
+            expect(body.article).toHaveLength(1)
```

Comment:
You can just expect article toEqual the article as it is stored in the database. Your test would also pass for a route of "/api/articles/2"